agent/api/task/task.go

```diff
@@ -322,6 +321,11 @@ func TaskFromACS(acsTask *ecsacs.Task, envelope *ecsacs.PayloadMessage) (*Task,
 		return nil, err
 	}
 
+	// Set the EnableFaultInjection field if present
+	if acsTask.EnableFaultInjection != nil {
+		task.EnableFaultInjection = aws.BoolValue(acsTask.EnableFaultInjection)
+	}
```

Comment:
nit: This is equivalent to 
```go
task.EnableFaultInjection = aws.BoolValue(acsTask.EnableFaultInjection)
```
as `aws.BoolValue` function takes care of `nil` handling. We might want to simplify the code.

Reply:
Fixed in [1d123a7](https://github.com/aws/amazon-ecs-agent/commit/1d123a79076c13fadefcb8e1af64efc3cd99beca)